src/frontend/babel.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  plugins: ["@emotion", transformImports()],
+  presets: ["next/babel"],
+};
+
+function transformImports() {
+  const materialUIPackages = ["core", "icons", "lab", "styles"];
+
+  const transform = {};
+
+  for (const package of materialUIPackages) {
```

Comment:
Material UI (MUI) has treeshaking for prod build, so we won't ship unneeded MUI code in prod mode. But we still need to transform imports to speed up dev mode build time! https://material-ui.com/guides/minimizing-bundle-size/#option-2